SDK/AppCenterCrashes/Microsoft.AppCenter.Crashes.Windows.Shared/Crashes.cs

```diff
@@ -135,5 +157,49 @@ public override bool InstanceEnabled
                 }
             }
         }
+
+        private void ProcessPendingErrors()
+        {
+            foreach (var file in ErrorLogHelper.GetErrorLogFiles())
+            {
+                AppCenterLog.Debug(LogTag, $"Process pending error file {file.Name}");
+                var log = ErrorLogHelper.ReadErrorLogFile(file);
+                if (log == null)
+                {
+                    AppCenterLog.Error(LogTag, $"Error parsing error log. Deleting invalid file: {file.Name}");
+                    try
+                    {
+                        file.Delete();
+                    }
+                    catch (System.Exception ex)
+                    {
+                        AppCenterLog.Warn(LogTag, $"Failed to delete error log file {file.Name}.", ex);
+                    }
+                }
+                else
+                {
+                    _unprocessedManagedErrorLogs.Add(log.Id, log);
+                }
+            }
+            SendCrashReportsOrAwaitUserConfirmation();
+        }
+
+        private void SendCrashReportsOrAwaitUserConfirmation()
+        {
+            HandleUserConfirmation();
```

Comment:
Why does this private method exist only to call another private method?  If it will be expanded later add a `TODO`

Reply:
This keeps the method structure as same as Android and this will be implemented in the next feature. It will be TODO but not for this implementation.